guides/source/association_basics.md

```diff
@@ -1278,36 +1486,31 @@ class Book < ApplicationRecord
 end
 ```
 
-TIP: In any case, Rails will not create foreign key columns for you. You need to explicitly define them as part of your migrations.
-
-##### `:foreign_type`
-
-Specify the column used to store the associated object’s type, if this is a polymorphic association. By default this is guessed to be the name of the association with a “`_type`” suffix. So a class that defines a `belongs_to :taggable, polymorphic: true` association will use “`taggable_type`” as the default `:foreign_type`.
+NOTE: Rails does not create foreign key columns for you. You need to explicitly define them in your migrations.
 
 ##### `:primary_key`
 
-By convention, Rails assumes that the `id` column is used to hold the primary key
-of its tables. The `:primary_key` option allows you to specify a different column.
+By default, Rails uses the `id` column as the primary key for its tables. The `:primary_key` option allows you to specify a different column as the primary key.
 
-For example, given we have a `users` table with `guid` as the primary key. If we want a separate `todos` table to hold the foreign key `user_id` in the `guid` column, then we can use `primary_key` to achieve this like so:
+For example, if the `users` table uses `guid` as the primary key instead of `id`, and you want the `todos` table to reference `guid` as a foreign key (`user_id`), you can configure it like this:
 
 ```ruby
 class User < ApplicationRecord
-  self.primary_key = 'guid' # primary key is guid and not id
+  self.primary_key = 'guid' # Sets the primary key to guid instead of id
 end
 
 class Todo < ApplicationRecord
-  belongs_to :user, primary_key: 'guid'
+  belongs_to :user, primary_key: 'guid' # References the guid column in users table
 end
 ```
 
-When we execute `@user.todos.create` then the `@todo` record will have its
-`user_id` value as the `guid` value of `@user`.
+When you execute `@user.todos.create`, the `@todo` record will have its `user_id` value set to the `guid` value of `@user`.
 
 ##### `:inverse_of`
```

Comment:
We already discuss `inverse_of` in-depth in the Bi-directional Association section.
Clear examples also require a `has_many` and a `belongs_to`, so I think it would be ok to remove it here?